org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/managers/GradleProjectImporterTest.java

```diff
@@ -65,6 +64,9 @@
 import com.google.common.collect.ImmutableList;
 import com.google.common.io.Files;
 
+import static org.eclipse.jdt.ls.core.internal.ProjectUtils.getJavaSourceLevel;
+import static org.eclipse.jdt.ls.core.internal.WorkspaceHelper.getProject;
+
```

Comment:
same here, should not change static imports order.

Reply:
Done.